app/controllers/api/v1/events_controller.rb

```diff
@@ -38,13 +39,15 @@ def update
 			address: params[:address],
 			location: params[:location],
 			is_private: params[:is_private]
-		)
-		render json: event[:id]
-
+			)
+			render json: event[:id]
+		else
+			halt(404)
+		end
+		render json: event
```

Comment:
Is this change from rendering the event ID to rendering the event going to cause any issues?